actionmailbox/test/dummy/config/environments/test.rb

```diff
@@ -46,4 +46,7 @@
 
   # Annotate rendered view with file names
   # config.action_view.annotate_rendered_view_with_filenames = true
+
+  # Raise error if unpermitted parameters are sent
+  config.action_controller.action_on_unpermitted_parameters = :raise
```

Comment:
I think we should add a new test to cover the bug being fixed here, rather than modify the configuration for the whole test suite.

You can change the behaviour for a single test by setting it on `ActionController::Parameters`:

```ruby
test "create inbound email with action_on_unpermitted_parameters = :raise" do
  original_action_on_unpermitted_parameters = ActionController::Parameters.action_on_unpermitted_parameters
  ActionController::Parameters.action_on_unpermitted_parameters = :raise
  # ...
ensure
  ActionController::Parameters.action_on_unpermitted_parameters = original_action_on_unpermitted_parameters
end
```

Reply:
My thinking when I suggested it was that it's better to run the whole test suite in as strict a configuration as possible. Then individual tests can opt out of the strictness, if they need it.

If the configuration had been set like this, the bug this PR fixes would never have slipped through, since there's already a test that covers it.

Reply:
I'm open to do this if desired, but this seems like it's indirectly testing that `action_on_unpermitted_parameters` works as intended, which should already be covered by that part of the suite. I could see submitting this as a test case that should not be committed to demonstrate the issue though.

I'm in agreement with @ghiculescu that the suite should be run with the stricter setting, which would have alerted the original author of the `create inbound email with attachments` test that it had this issue.

I am happy to revert the config change in this PR, but as you can see, there are no other tests in the suite that fail with it set to `:raise`, and this offers protection for future contributors.

For some personal background, several applications I have worked with also have `action_on_unpermitted_parameters = :raise` set in development, because the default behavior of `:log` has caused developers on my teams to miss issues that were only later caught in production.

Reply:
To add to this, when we added and enabled ActionMailbox in our application, we did so specifically to process inbound email attachments, so we were quite confused when we were unable to send a test email that included one from the conductor, and had to dig into Rails internals to discover the issue.

We temporarily worked around it with a monkey-patch in our app, before realizing that it is related to this config setting, which I admit is nonstandard, but I believe should only blow up when making an error in our application, which would be easier to track down, and not for Rails internals, which should work in every configuration, if possible.